docs/proposals/keadm-debug.md

```diff
@@ -0,0 +1,316 @@
+  5. Use command `disk` check whether the disk meets the requirements, requirements minimum 1 GB
+  6. Use command `dns` Check whether the node domain name resolution function is normal
+  7. Use command `runtime `  Check whether the node container runtime function is installed, can use parameter `--runtime` to set container runtime,  default is docker
+  8. Use command `network `  check whether the node can communicate with the endpoint on the cloud,    can use parameter `--ip` to set test ip, default to ping clusterdns
+  9. Use command `pid ` check if the current number of processes in the environment is too many. If the number of available processes is less than 5%, the number of processes is considered insufficient
+
+`keadm debug collect`
+
+- What is it?
+
+  - This command will obtain all related data of the current node, and then locate and use  operation personnel
+
+- What shall be its scope ?
+
+  1. system data
+
+    - Hardware architecture
+
+      Collect arch command output and determine the type of  installation
+
+    - CPU information
+
+      Parse the /proc/cpuinfo file and output the cpu information file
+
+    - Memory information
+
+      Collect free -h command output
+
+    - Hard disk information
+
+      Collect df -h command output, and mount command output
+
+    - Internet Information
+
+      Collect netstat -anp command output and copy /etc/resolv.conf and /etc/hosts files
+
+    - Process information
+
+      Collect ps -aux command output
+
+    - Time information
+
+      Collect date and uptime command output
+
+    - History command input
+
+      Collect all the commands entered by the current user
+
+  2. Edgecore data
+
+  - database data
+
+    Copy the /var/lib/kubeedge/edgecore.db file
+
+  - log files
+
+    Copy all files under /var/*log*/*kubeedge*/
+
+  - service file
+
+    Copy the edgecore.service files under /lib/systemd/system/
+
+  - software version
+
+  - certificate
+
+    Copy all files under /etc/kubeedge/certs/
+
+  - Edge-Core configuration file in  software
+
+    Copy all files under /etc/kubeedge/config/
+
+  3. Container runtime data
+
+  - runtime version information
+
+  - runtime container information
+
+  - runtime log information
+
+  - runtime configuration and log information
+
+  - runtime image information
+
+`keadm debug get`
+
+- What is it?
+  
+  - This command will get and format the specified resource`s information from the local database of the edge node
+  
+- What shall be its scope ?
+
+  1. Format resource information from the local database, and available resource types:
+    - `pod`
+    - `node`
+    - `service`
+    - `secret`
+    - `configmap`
+    - `endpoint`
+  2. Use flag `-n, --namespace=''` to indicate the scope of resource acquisition, if the flag `-A, --all-namespaces` is used, information of the specified resource will be obtained from all ranges
```

Comment:
1. add more options such as edgecore path that will be visited, default to `/var/lib/kubeedge/edgecore.db`
2. detailing the  option `-o` ,what formats we support?
- yaml
- json
- **wide**

Reply:
Thank you for your suggestions for optimization，it will be modified soon